extensions/prototypes/labgraph_monitor/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
What is this robots.txt used for?

Reply:
robots.txt file tells search engine crawlers which URLs the crawler can access on your site(Robots Exclusion Protocol). This is used mainly to avoid overloading the site with requests. It is mainly useful in case Labgraph Monitor will be deployed. It can be deleted in case of localhost usage.